src/main/java/dbProcs/Setter.java

```diff
@@ -924,4 +924,36 @@ public static boolean userCreate (String ApplicationRoot, String classId, String
 		log.debug("*** END userCreate ***");	
 		return result;
 	}
+	
+	public static boolean userDelete (String ApplicationRoot, String userId) throws SQLException

```

Comment:
This method needs an associated JUnit Test for the Definition of Done

Reply:
Added associated JUnit test method in src/test/java/dbProcs/SetterTest.java

Reply:
Looks Good!